src/components/ReportWelcomeText.tsx

```diff
@@ -190,7 +190,7 @@ function ReportWelcomeText({report, policy}: ReportWelcomeTextProps) {
                         <Text>{welcomeMessage.phrase1}</Text>
                     </Text>
                 )}
-                {isDefault && (
+                {isDefault && displayNamesWithTooltips.length && (
```

Comment:
@nkdengineer I think this line is causing UI to show `0` for test step `4. Create a group chat with only yourself`, instead of showing nothing:

<img width="966" alt="Screenshot 2024-10-08 at 15 12 55" src="https://github.com/user-attachments/assets/c05a6c8d-b020-4569-9025-9cdcf7d0268b">

Reply:
@ikevin127 We decided to remove it in this case. The context [here](https://github.com/Expensify/App/issues/47427#issuecomment-2400697180).

Reply:
@nkdengineer I understand that, but the problem is that with the current code there's a `0` showing up instead of nothing (take a closer look at the picture.

Reply:
@ikevin127 I updated to fix it now.